akka-docs/src/main/paradox/typed/actor-discovery.md

```diff
@@ -29,7 +29,7 @@ applicable.
 ## Receptionist
 
 When an actor needs to be discovered by another actor but you are unable to put a reference to it in an incoming message,
-you can use the `Receptionist`. It supports both local and cluster(see @ref:[cluster](#cluster-receptionist)). You register the specific actors that should be discoverable 
+you can use the @apidoc[receptionist.Receptionist]. It supports both local and cluster(see @ref:[cluster](#cluster-receptionist)). You register the specific actors that should be discoverable 
 from each node in the local `Receptionist` instance. The API of the receptionist is also based on actor messages. 
 This registry of actor references is then automatically distributed to all other nodes in the case of a cluster. 
 You can lookup such actors with the key that was used when they were registered. The reply to such a `Find` request is 
```

Comment:
`Find`, `Listing`, `Subscribe` don't belong to `Receptionist`. `ReceptionistMessages` contains them, but it is private.